tests/e2e/util.go

```diff
@@ -941,6 +941,31 @@ func updateCSIDeploymentTemplateFullSyncInterval(client clientset.Interface, min
 	return oldValue
 }
 
+// updateCSIDeploymentProvisionerTimeout helps to update the timeout value
+// in deployment template. For this to take effect,
+func updateCSIDeploymentProvisionerTimeout(client clientset.Interface, namespace string, timeout string) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	deployment, err := client.AppsV1().Deployments(namespace).Get(
+		ctx, vSphereCSIControllerPodNamePrefix, metav1.GetOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	containers := deployment.Spec.Template.Spec.Containers
+	for _, c := range containers {
+		if c.Name == "csi-provisioner" {
+			for i, e := range c.Args {
+				if strings.Contains(e, "timeout") {
+					c.Args[i] = "--timeout=" + timeout + "s"
```

Comment:
check here if the timeout is already set to correct value, in which case just return from here...

Reply:
Fixed